update_presenze.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python3
+import sys
+
+__author__ = 'Amedeo Zampieri'
+
+fin = open('presenze.md', 'r')
```

Comment:
In python3, sarebbe più ideomatico usare il costrutto with:

```python
with open('presenze.md', 'r') as fin:
    # uso fin
```

Idem per gli altri file che vengono aperti (es. `fnomi_presenti` non viene chiuso)